dockerfiles/alpine/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN set -e; for pkg in $(go list ./...); do \
 
 FROM alpine:edge AS resource
 RUN apk add --no-cache bash tzdata ca-certificates unzip zip gzip tar
-COPY --from=builder assets/ /opt/resource/
+COPY --from=builder /assets/ /opt/resource/
```

Comment:
quick note: buildkit has fixed the compatibility issue here via https://github.com/moby/buildkit/pull/1125 - though probably better to stick with absolute paths anyway